testplan/web_ui/testing/package.json

```diff
@@ -16,13 +16,15 @@
     "enzyme-adapter-react-16": "1.6.0",
     "eslint": "5.10.0",
     "eslint-plugin-react": "7.11.1",
+    "npm": "^6.10.1",
```

Comment:
This npm dependency was added automatically?

Reply:
I could have done this when I ran npm install? I did not add this manually

Reply:
Can you please remove this actually. It seems to cause problems when installing internally, I don't think it's needed to specify npm as a dependency.